synapse/lib/cell.py

```diff
@@ -3941,25 +3942,39 @@ async def addSignalHandlers(self):
         await s_base.Base.addSignalHandlers(self)
 
         def sighup():
-            asyncio.create_task(self.reloadCell())
+            asyncio.create_task(self.reload())
 
         loop = asyncio.get_running_loop()
         loop.add_signal_handler(signal.SIGHUP, sighup)
 
-    def addReloadFunc(self, name: str, func):
+    def addReloadSystem(self, name: str, func):
+        '''
+        Add a reload system. This may be dynamically called at at time.
+
+        Args:
+            name (str): Name of the function.
+            func: The callable for reloading a given system.
+
+        Note:
+            Reload functions take no arguments when they are executed.
+            Values returned by the reload function must be msgpack friendly.
+
+        Returns:
+            None
+        '''
```

Comment:
This whole comment is naming pedantry so feel free to ignore ...

```suggestion
    def addReloadableSystem(self, name: str, func: callable):
        '''
        Add a reloadable system. This may be dynamically called at at time.

        Args:
            name (str): Name of the system.
            func: The callable for reloading a given system.

        Note:
            Reload functions take no arguments when they are executed.
            Values returned by the reload function must be msgpack friendly.

        Returns:
            None
        '''
```